### PR TITLE
Add Filters to Housing

### PR DESCRIPTION
**Pull request by mboperator**
screenshot from 2017-12-29 14-47-53
screenshot from 2017-12-29 14-47-47

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -33,6 +33,7 @@ const store = createStore(s => s, {}, install());
 const AppContainer = styled.div`
   width: 100vw;
   height: 100vh;
+  overflow-x: hidden;
```
Comment (jeyip) [inline src/App.js:36]: 🙌

File: src/components/HouseCard.jsx [hunk 2]
```diff
@@ -49,14 +49,12 @@ const HeaderContainer = styled.div`
 
 const Button = styled.button`
   align-self: flex-end;
-  background-color: ${props => props.active ? 'darkgray' : 'none'};
-  color: ${props => props.active ? 'white' : 'black'};
+  background-color: ${props => props.active ? '#3A3A3A' : '#FFF'};
+  color: ${props => props.active ? '#000' : '#FFF'};
```
Comment (jeyip) [inline src/components/HouseCard.jsx]: Do we want `color` of text to be white when unactive?

File: src/components/MultiSelect.jsx [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+const Container = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: row;
+`
+
+const Button = styled.div`
+  border: 1px solid white;
+  padding: 5px 10px;
+  color: ${({ selected }) => selected ? 'black' : 'white'};
```
Comment (jeyip) [inline src/components/MultiSelect.jsx]: `NB:` hex codes for consistency

File: src/modules/housing.js [hunk 4]
```diff
@@ -34,51 +34,40 @@ const serializeForCreate = params => {
     email_address: params.emailAddress,
   };
 };
+/*
```
Comment (jeyip) [inline src/modules/housing.js]: `NB:` Are we keeping these comments?

File: src/components/atoms.jsx [hunk 5]
```diff
@@ -1,6 +1,37 @@
+import React from 'react'
 import styled, { keyframes } from 'styled-components'
 import { Link } from 'react-router-dom'
 
+const Label = styled.div`
+  font-size: 1.5rem;
```
Comment (jeyip) [inline src/components/atoms.jsx]: `NB:` Are we still using rem?

File: src/modules/housing.js (hunk 4 above)
Comment (mboperator) [reply src/modules/housing.js]: Great question -- I'll remove these.

File: src/components/atoms.jsx (hunk 5 above)
Comment (mboperator) [reply src/components/atoms.jsx]: Nope! I believe this was just copy-pasta'd from another Label. I'll switch this up

**Review by jeyip — APPROVED** [review]

File: src/components/HouseCard.jsx (hunk 2 above)
Comment (mboperator) [reply src/components/HouseCard.jsx]: I think I inherited a lot of these changes from your branch... Do we?

Comment (jeyip) [reply src/components/HouseCard.jsx]: Switch it up or we can fix it in another PR. Let's get this out ASAP

**Pull request merged**